spec/defines/dns_dnssec_policy_spec.rb

```diff
@@ -0,0 +1,92 @@
+require 'spec_helper'
+
+describe 'dns::dnssec_policy' do
+  on_supported_os.each do |os, os_facts|
+    context "on #{os}" do
+      let(:facts) { os_facts }
+      let(:title) { "standard" }
+
+      let :pre_condition do
+        'include dns'
+      end
+
+      context "without params" do
+        it { is_expected.to compile }
+
+        it "should have valid zone file contents" do
+          verify_concat_fragment_exact_contents(catalogue, 'dnssec-policy-standard', [
+            'dnssec-policy standard {',
+            '};',
+          ])
+        end
+
+        context "when using a reserved name" do
+          let(:title) { "default" }
+
+          it { is_expected.not_to compile }
```

Comment:
I prefer to check the error message too:
```suggestion
          it { is_expected.to compile.and_raise_error(/is reserved/) }
```